### PR TITLE
[Docs]Add connector release status

### PR DESCRIPTION
**Pull request by CalvinKirs**
Add the health state description of the connector
In addition, we should add the release date, status change version, function addition description on each connector page

Hi, if you have tested or used the connector in production, please leave a comment below this PR describing its specific status and I will update it based on your description.

### CODE REVIEW

File: docs/en/connector-v2/connector-health-status.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+| Production Readiness | No                                                                                                                                                                                                               | Yes                                                                                                                                                                                                                                        | Yes                                                                                                                                                                                                            |
+
+## Connector V2 Health
+
+| Connector Name    | Type   | Status | Support Version |
+|-------------------|--------|--------|----------------|
+| Asset             | Sink   | Beta   | 2.2.0-beta     |
```
Comment (Hisoka-X) [inline docs/en/connector-v2/connector-health-status.md]: Suggest add link to connector detail page.

**Comment by EricJoy2048** [comment]
Please update Hive Source Connector and Hive Sink Connector to `Alpha`

File: docs/en/connector-v2/connector-health-status.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+| Jdbc              | Sink   | Beta   | 2.2.0-beta     |
+| Kudu              | Source | Alpha   | 2.2.0-beta     |
+| Kudu              | Sink   | Alpha   | 2.2.0-beta     |
+| LocalFile         | Sink   | Beta   | 2.2.0-beta     |
+| LocalFile         | Source | Beta   | 2.2.0-beta     |
+| MongoDB           | Source | Alpha  | 2.2.0-beta     |
+| MongoDB           | Sink   | Alpha  | 2.2.0-beta     |
+| Neo4j             | Sink   | Alpha  | 2.2.0-beta     |
+| OssFile           | Sink   | Alpha   | 2.2.0-beta     |
+| OssFile           | Source | Alpha   | 2.2.0-beta     |
```
Comment (Hisoka-X) [inline docs/en/connector-v2/connector-health-status.md]:
```suggestion
| OssFile           | Source | Beta   | 2.2.0-beta     |
```

File: docs/en/connector-v2/connector-health-status.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+| Connector Name    | Type   | Status | Support Version |
+|-------------------|--------|--------|----------------|
+| Asset             | Sink   | Beta   | 2.2.0-beta     |
+| ClickHouse        | Source | Beta   | 2.2.0-beta     |
+| ClickHouse        | Sink   | Beta   | 2.2.0-beta     |
+| ClickHouseFile    | Sink   | Beta   | 2.2.0-beta     |
+| Console           | Source | Beta   | 2.2.0-beta     |
+| DataHub           | Sink   | Alpha  | 2.2.0-beta     |
+| Elasticsearch     | Sink   | Alpha  | 2.2.0-beta     |
```
Comment (Hisoka-X) [inline docs/en/connector-v2/connector-health-status.md]:
```suggestion
| Elasticsearch     | Sink   | Beta  | 2.2.0-beta     |
```

File: docs/en/connector-v2/connector-health-status.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+| FtpFile           | Sink   | Alpha  | 2.2.0-beta     |
+| FtpFile           | Source | Alpha  | 2.2.0-beta     |
+| Greenplum         | Sink   | Alpha  | 2.2.0-beta     |
+| Greenplum         | Source | Alpha  | 2.2.0-beta     |
+| HdfsFile          | Sink   | Beta   | 2.2.0-beta     |
+| HdfsFile          | Source | Beta   | 2.2.0-beta     |
+| Hive              | Sink   | Beta   | 2.2.0-beta     |
+| Hive              | Source | Beta   | 2.2.0-beta     |
+| Http              | Sink   | Alpha  | 2.2.0-beta     |
+| Http              | Source | Alpha  | 2.2.0-beta     |
```
Comment (hailin0) [inline docs/en/connector-v2/connector-health-status.md]:
```suggestion
| Http              | Sink   | Beta   | 2.2.0-beta     |
| Http              | Source | Beta   | 2.2.0-beta     |
```

File: docs/en/connector-v2/connector-health-status.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+| IotDB             | Source | Beta   | 2.2.0-beta     |
+| IotDB             | Sink   | Beta   | 2.2.0-beta     |
+| Jdbc              | Source | Beta   | 2.2.0-beta     |
+| Jdbc              | Sink   | Beta   | 2.2.0-beta     |
+| Kudu              | Source | Alpha   | 2.2.0-beta     |
+| Kudu              | Sink   | Alpha   | 2.2.0-beta     |
+| LocalFile         | Sink   | Beta   | 2.2.0-beta     |
+| LocalFile         | Source | Beta   | 2.2.0-beta     |
+| MongoDB           | Source | Alpha  | 2.2.0-beta     |
+| MongoDB           | Sink   | Alpha  | 2.2.0-beta     |
```
Comment (hailin0) [inline docs/en/connector-v2/connector-health-status.md]:
```suggestion
| MongoDB           | Source | Beta   | 2.2.0-beta     |
| MongoDB           | Sink   | Beta   | 2.2.0-beta     |
```

File: docs/en/connector-v2/connector-health-status.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+| Neo4j             | Sink   | Alpha  | 2.2.0-beta     |
+| OssFile           | Sink   | Alpha   | 2.2.0-beta     |
+| OssFile           | Source | Alpha   | 2.2.0-beta     |
+| Phoneix           | Sink   | Alpha   | 2.2.0-beta     |
+| Phoneix           | Source | Alpha   | 2.2.0-beta     |
+| Pulsar            | Source | Beta   | 2.2.0-beta     |
+| Redis             | Sink   | Alpha   | 2.2.0-beta     |
```
Comment (ashulin) [inline docs/en/connector-v2/connector-health-status.md]:
```suggestion
| Redis             | Sink   | Beta   | 2.2.0-beta     |
```

**Review by hailin0 — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]
+1

**Review by TyrantLucifer — APPROVED** [review]
+1

**Pull request merged**